torch/lib/TH/THGeneral.c

```diff
@@ -359,25 +359,27 @@ TH_API void THInferNumThreads(void)
 #endif
 }
 
-TH_API THDescBuff _THSizeDesc(const long *size, const long ndim) {
-  const int L = TH_DESC_BUFF_LEN;
+TH_API THDescBuff _THSizeDesc(const uint8_t *size, const uint8_t ndim) {
```

Comment:
Most minor bugs like this are fixed in newer commit [Fixed some bugs that prevent build failure on Windows](https://github.com/pytorch/pytorch/pull/2402/commits/a4fc05a5a2feba376658b333a4f532ee9f5073d9).